edge/pkg/metamanager/process.go

```diff
@@ -485,8 +470,6 @@ func (m *metaManager) process(message model.Message) {
 		m.processQuery(message)
 	case model.ResponseOperation:
 		m.processResponse(message)
-	case messagepkg.OperationNodeConnection:
```

Comment:
Also can remove the code on the sender side?

Reply:
done